glusterhealth/reports/res-memusage-glusterd.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2017 Red Hat, Inc.
+#
+# This file is part of gluster-health-report project which is a
+# subproject of GlusterFS ( www.gluster.org)
+#
+# This file is licensed to you under your choice of the GNU Lesser
+# General Public License, version 3 or any later version (LGPLv3 or
+# later), or the GNU General Public License, version 2 (GPLv2), in all
+# cases as published by the Free Software Foundation.
+
+import logging
+
+from utils import command_output, CommandError
+
+#TODO:We need to extend this to glusterfsd and glusterfs.
+procs = ['glusterd']
+
+def report_resmemusagegluster(ctx):
+    try:
+
+        for i in procs:
+            cmd = "cat /proc/`pgrep %s`/status | grep -i vmrss | awk '{print $2}'" % (i)
+            out = command_output(cmd)
+            ctx.ok("Glusterd ResMem in kB", memory=out.strip())
+
```

Comment:
I expect to have some suggestions on what is the ideal memory usage? Say anything below 1GB is in OK range. anything with >1GB -- 3GB is 'warn', and above '3GB'  as notok?